### PR TITLE
Add CI steps

### PR DESCRIPTION
**Pull request by emnigma**
- replace argparse config with configparse
- refactor code so train params can be extracted into `config.ini`
- remote transitive deps from `requirements.txt`
- add working CI script

### CODE REVIEW

File: .github/workflows/build_and_run.yaml [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11.x'
+
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
           submodules: true
+          lfs: true
```
Comment (gsvgit) [inline .github/workflows/build_and_run.yaml:18]: Why we use LFS?

File: .github/workflows/build_and_run.yaml [hunk 2]
```diff
@@ -24,9 +35,21 @@ jobs:
...
+      - name: Move SerializedEpisodes
+        working-directory: ./AIAgent
+        run: |
+          mkdir report
+          mv ../GameServers/VSharp/VSharp.ML.GameServer.Runner/bin/Release/net7.0/8100/SerializedEpisodes/ report/
 
       - name: Launch application
+        working-directory: ./AIAgent
+        run: |
+          python3 launch_servers.py &
```
Comment (gsvgit) [inline .github/workflows/build_and_run.yaml:49]: Looks like ```launch_servers``` also should be configurable. At least, one can specify number of servers.

Comment (emnigma) [reply .github/workflows/build_and_run.yaml:49]: Lets do it in another issue

File: .github/workflows/build_and_run.yaml (hunk 1 above)
Comment (emnigma) [reply .github/workflows/build_and_run.yaml:18]: - [ ] remove

File: .github/workflows/build_and_run.yaml (hunk 2 above)
Comment (emnigma) [reply .github/workflows/build_and_run.yaml:48]: mentioned in #4

**Pull request merged**